tools/util/include/cutlass/util/reference/host/tensor_fill.h

```diff
@@ -404,7 +429,7 @@ void TensorFillRandomGaussian(
 }
 
 ///////////////////////////////////////////////////////////////////////////////////////////////////
-/// Fills a tensor with random values with a Gaussian distribution.
+/// Fills a rank-2 tensor symmetrically with random values of a Gaussian distribution.
```

Comment:
This is filling up a "symmetric rank-2 tensor", not filling up a "rank-2 tensor symmetrically" :). Bcoz for a symmetric matrix we don't fill up the invalid region.